sage-ios/SAGE/SAGE/UIFont.swift

```diff
@@ -23,7 +23,7 @@ extension UIFont {
     @nonobjc static let titleFont = UIFont.getDefaultFont(17)
     
     static func getDefaultFont(size: CGFloat = 20) -> UIFont {
-        if size < 20 {
+        if size < 17 {
```

Comment:
made these changes cuz i think these are the right numbers actually..irrelevant to rest of PR